_csharp-api/inputsystem/input.md

```diff
@@ -2,8 +2,60 @@
 
 This static type provides you with input functionality, the base input system has no events to bind to and supports polling only. It's largely self explanatory with a few possible gotchas.
 
-## Key Polling
+## Key Reading
 
-`GetKeyDown(KeyCode)` | `GetKeyPressed(KeyCode)` | `GetKeyReleased(KeyCode)` - These are likely to be the most commonly used methods, `GetKeyDown` will return true on any number of frames so long as a key is down, this can cause undesired behaviour. If you only want to read an input once, use `GetKeyPressed` or `GetKeyReleased`.
+There's a number of usefull methods that detect keys being pressed down in different ways, all seeming to return a 'bool'. Most common are `isKeyDown`, `isKeyPressed` and `isKeyReleased`.
 
-You can also use the extension methods `IsDown`/`IsPressed`/`IsReleased` like so `KeyCode.A.IsPressed()`
\ No newline at end of file
+You may also use the extension methods `isDown`, `isPressed` and `isReleased` as *extension* alternatives of the above
+
+Below, you may find an in-depth documentation of each.
+
+ ### Input.IsKeyDown(InputKey.[key])
+        This checks to see if the specified [key] is currently being pressed, it returns true as long as the [key] is held down. 
+        Example:
+
+        if(Input.IsKeyDown(InputKey.Y)){
```

Comment:
Put code in ``, please.

Reply:
Formating on GitHub is above my level of intelligence. For whatever reason everything under the header looks like they are inside "`" and adding `` doesn't further format.

Reply:
```
Some code
```

Reply:
And remove tabs after headers. It makes quotes.
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes

Reply:
I quite literally just spend 30 minutes and 10 attempts and `` is not a thing, finally worked with ```

FIXED, at last